### ISSUE TITLE
crashes when i try to refresh contact list

### ISSUE THREAD

**Issue by hugomrdias** [Bajoja/indicator-kdeconnect#73]
```
Exception in thread Sync:
Traceback (most recent call last):
  File "/usr/share/indicator-kdeconnect/Sms.py", line 161, in get_contacts
    space_requests, connections = get_connections()
  File "/usr/share/indicator-kdeconnect/Sms.py", line 144, in get_connections
    response = google.get(people_api, params=params_paging).json()
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 480, in get
    return self.request('GET', url, **kwargs)
  File "/usr/lib/python3/dist-packages/requests_oauthlib/oauth2_session.py", line 235, in request
    http_method=method, body=data, headers=headers)
  File "/usr/lib/python3/dist-packages/oauthlib/oauth2/rfc6749/clients/base.py", line 194, in add_token
    raise TokenExpiredError()
oauthlib.oauth2.rfc6749.errors.TokenExpiredError: (token_expired) 

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.5/threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/share/indicator-kdeconnect/Sms.py", line 164, in get_contacts
    space_requests, connections = get_connections()
  File "/usr/share/indicator-kdeconnect/Sms.py", line 144, in get_connections
    response = google.get(people_api, params=params_paging).json()
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 480, in get
    return self.request('GET', url, **kwargs)
  File "/usr/lib/python3/dist-packages/requests_oauthlib/oauth2_session.py", line 235, in request
    http_method=method, body=data, headers=headers)
  File "/usr/lib/python3/dist-packages/oauthlib/oauth2/rfc6749/clients/base.py", line 194, in add_token
    raise TokenExpiredError()
oauthlib.oauth2.rfc6749.errors.TokenExpiredError: (token_expired)

```

latest mint 
latest indicator-kdeconnect

**Comment by hugomrdias** [comment]
also i can't get it to ask me again for auth 
even tried remove purge and nothing

**Comment by Bajoja** [comment]
@hugomrdias - Go to "/home/{USERNAME}/.local/share/indicator-kdeconnect/sms" delete everything and try again.

@OzymandiasTheGreat - Token have expiration date?

**Comment by OzymandiasTheGreat** [comment]
Yeah, tokens expire after 1 hour iirc. However tokens should be refreshed automatically, I'm looking into this.

**Comment by Bajoja** [comment]
@OzymandiasTheGreat OK

**Comment by OzymandiasTheGreat** [comment]
Apparently I was using old token even after automatic token refresh. Still, it should've just worked on restart.

**Comment by Bajoja** [comment]
@hugomrdias fixed on 02e353d
